modules/util/portal-tools-rest-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/rest/builder/test/resource/v1_0/test/SiteTestEntityResourceTest.java

```diff
@@ -61,6 +62,59 @@ public void testGetSiteSiteTestEntitiesPage() throws Exception {
 			testGroup.getGroupKey(), totalCount + 2);
 	}
 
+	@Override
+	@Test
+	@TestInfo("LPD-54012")
+	public void testGraphQLGetSiteSiteTestEntitiesPage() throws Exception {
+		JSONObject siteTestEntitiesJSONObject = JSONUtil.getValueAsJSONObject(
+			invokeGraphQLQuery(
+				new GraphQLField(
+					"siteTestEntities",
+					HashMapBuilder.<String, Object>put(
+						"siteKey",
+						"\"" + testGroup.getExternalReferenceCode() + "\""
+					).build(),
+					new GraphQLField("items", getGraphQLFields()),
+					new GraphQLField("page"), new GraphQLField("totalCount"))),
+			"JSONObject/data", "JSONObject/siteTestEntities");
+
+		long totalCount = siteTestEntitiesJSONObject.getLong("totalCount");
+
+		testGraphQLGetSiteSiteTestEntitiesPage_addSiteTestEntity();
+
+		siteTestEntitiesJSONObject = JSONUtil.getValueAsJSONObject(
+			invokeGraphQLQuery(
+				new GraphQLField(
+					"siteTestEntities",
+					HashMapBuilder.<String, Object>put(
+						"siteKey", "\"" + testGroup.getGroupId() + "\""
+					).build(),
+					new GraphQLField("items", getGraphQLFields()),
+					new GraphQLField("page"), new GraphQLField("totalCount"))),
+			"JSONObject/data", "JSONObject/siteTestEntities");
+
+		Assert.assertEquals(
+			totalCount + 1, siteTestEntitiesJSONObject.getLong("totalCount"));
+
+		testGraphQLGetSiteSiteTestEntitiesPage_addSiteTestEntity();
+
+		siteTestEntitiesJSONObject = JSONUtil.getValueAsJSONObject(
+			invokeGraphQLQuery(
+				new GraphQLField(
+					"siteTestEntities",
+					HashMapBuilder.<String, Object>put(
+						"siteKey", "\"" + testGroup.getGroupKey() + "\""
+					).build(),
+					new GraphQLField("items", getGraphQLFields()),
+					new GraphQLField("page"), new GraphQLField("totalCount"))),
+			"JSONObject/data", "JSONObject/siteTestEntities");
+
+		Assert.assertEquals(
+			totalCount + 2, siteTestEntitiesJSONObject.getLong("totalCount"));
+
+		super.testGraphQLGetSiteSiteTestEntitiesPage();
```

Comment:
I think in general, we call the parent method as the first instruction, then it is more clear that is being executed